python/paddle/fluid/dygraph/dygraph_to_static/ast_transformer.py

```diff
@@ -14,7 +14,7 @@
 
 from __future__ import print_function
 
-import ast
+import gast
```

Comment:
This PR LGTM overall.

But in next PR we had better to add a web link to gast as comment in this .py file so that future developers can easily find gast grammar tree